src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -2135,6 +2234,22 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 	}
 }
 
+function setupMultiselectDragLabel(text: string, dataTransfer: DataTransfer, tab: HTMLElement) {
```

Comment:
We have a utility method `dnd.ts#applyDragImage()`, could we use it here?

Reply:
Go to know, we use it!